### ISSUE TITLE
Hi, single detection task, the mAP is very low,what's wrong?

### ISSUE THREAD

**Issue by zjsun7** [valeoai/WoodScape#60]
For the single detection task, the mAP is very low and does not reach the mAP63 mentioned in the paper. May I ask what is wrong?

the train setting: single detection task, train_file is train.txt, val_file is val.txt, input height is 576, input_width is 1088, num_classes_detection is 5, batch_size 24, epoch 20, learning_rate 0.0001, models_to_load: encoder and detection, pretrained_weihts is res50, use two GPU of  48G, the frame_idxs is [0], the other setting is same.

the result is：mAP 0.199 at step 3000 on 8 epoch, mAP per class in order [0.303, 0.265, 0.17, 0.105, 0.154]. The final loss is 1.26.

Looking forward to your reply, thanks!

**Comment by rvarun7777** [comment]
Hi,

Sorry to comment the same again. Thanks for trying out the dataset and the code. Unfortunately, you can not reach the performance of the paper for the reasons stated in the [README](https://github.com/valeoai/WoodScape/blob/63554efe0f2df46a409cb6cc8b6893b3dc3dea2f/omnidet/README.md).
